### PR TITLE
Release v0.32.0

### PR DESCRIPTION
**Pull request by jchengsfx**
Requires #4614

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,47 @@
...
+  - New implementation of an in-memory queue to store traces, data compression with gzip, and queue configuration options (#4395)
+  - Make `Hclog2ZapLogger` struct and methods private for public go api review (#4431)
+- `newrelic` exporter (#4392):
+  - Marked unsupported metric as permanent error
+  - Force the interval to be valid even if 0
+- `awsxray` exporter: Add PHP stacktrace parsing support (#4454)
+- `file_storage` extension: Implementation of batch storage API (#4145)
+- `alibabacloudlogservice` exporter: Update labels to attributes for OTLP 0.9.0 (#4588)
+- `dynatrace` exporter: Update labels to attributes for OTLP 0.9.0 (#4590)
+- `redis` receiver: Move `interval` runner package to `internal/interval` (#4600)
```
Comment (dmitryax) [inline CHANGELOG.md:27]: this change is no-op, let's remove it

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,47 @@
...
+- `logzio` exporter:
+  - New implementation of an in-memory queue to store traces, data compression with gzip, and queue configuration options (#4395)
+  - Make `Hclog2ZapLogger` struct and methods private for public go api review (#4431)
+- `newrelic` exporter (#4392):
+  - Marked unsupported metric as permanent error
+  - Force the interval to be valid even if 0
+- `awsxray` exporter: Add PHP stacktrace parsing support (#4454)
+- `file_storage` extension: Implementation of batch storage API (#4145)
+- `alibabacloudlogservice` exporter: Update labels to attributes for OTLP 0.9.0 (#4588)
+- `dynatrace` exporter: Update labels to attributes for OTLP 0.9.0 (#4590)
```
Comment (dmitryax) [inline CHANGELOG.md:26]: let's remove all the _labels to attributes_ changes. those are internal changes not affecting end user

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,47 @@
...
+- `signalfx` exporter:
+  - Add process metric to translation rules (#4598)
+  - Update labels to attributes for OTLP 0.9.0 (#4603)
+
+## 🧰 Bug fixes 🧰
+
+- `awsxray` exporter: Fix the origin on ECS/EKS/EB on EC2 cases (#4391)
+- `splunk_hec` exporter:
+  - Prevent re-sending logs that were successfully sent (#4467)
+  - Send profiling data separately from logs (#4464)
```
Comment (dmitryax) [inline CHANGELOG.md:43]: This is not a bug fix but rather enhancement. I'd call it "Add support for profiling logs"

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,38 @@
...
+
+- `logzio` exporter:
+  - New implementation of an in-memory queue to store traces, data compression with gzip, and queue configuration options (#4395)
+  - Make `Hclog2ZapLogger` struct and methods private for public go api review (#4431)
+- `newrelic` exporter (#4392):
+  - Marked unsupported metric as permanent error
+  - Force the interval to be valid even if 0
+- `awsxray` exporter: Add PHP stacktrace parsing support (#4454)
+- `file_storage` extension: Implementation of batch storage API (#4145)
+- `alibabacloudlogservice` exporter: Update labels to attributes for OTLP 0.9.0 (#4588)
```
Comment (dmitryax) [inline CHANGELOG.md:25]: One more left

**Comment by tigrannajaryan** [comment]
This needs an update since several other PRs were merged today.

**Comment by jchengsfx** [comment]
> This needs an update since several other PRs were merged today.

Done.

**Review by dmitryax — APPROVED** [review]

**Comment by tigrannajaryan** [comment]
This needs to wait for https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/4614

**Comment by jchengsfx** [comment]
Also should wait for #4629

**Pull request merged**